STYLE.md

```diff
@@ -268,6 +268,11 @@ make sense to have the same name, while avoiding shadowing the outer variable.
 		return this.processResult(result);
 	});
 
+	// right
+	promise.then(function (result) => {
```

Comment:
You're right! Sorry about that leftover from copy and paste from the example above. Thanks for catching that. :-)